app/src/main/java/org/systers/mentorship/view/fragments/EditProfileFragment.kt

```diff
@@ -77,11 +83,11 @@ class EditProfileFragment: DialogFragment() {
 
         val editProfileDialog = dialog as AlertDialog
         editProfileDialog.getButton(AlertDialog.BUTTON_POSITIVE).setOnClickListener {
-            //TODO: Construct the object newUser by getting the new information
-            if (newUser == user) {
-                dismiss()
+            // TODO: Add validation
```

Comment:
@m-murad What sort of validation? I just need to know so that I can create an issue later for this.

Reply:
Check is name is valid or has the user provided an empty string.